resources/js/Layouts/ApplicationLayout/ApplicationLayout.tsx

```diff
@@ -147,7 +147,7 @@ const ApplicationLayout = ({children}: LayoutProps) => {
                         justifyContent='space-evenly'
                         alignItems='center'>
                         <Grid item container xs md={6} justifyContent='center'>
-                            <Username>{getUsername()}</Username>
+                            <Username id='user'>{getUsername()}</Username>
```

Comment:
not a fan of this, can you look into `data-cy` attribute, you can add it here 
look at the best practice here 

https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
This helps a lot, thank you! I'll change it an re-request your approval.